### DISCUSSION TITLE
Layout for Repositories

### DISCUSSION THREAD

**Discussion by hlsantos** [community/community#23804] [New to GitHub]
<p>I am basically new to GitHub.  We have been using CVS for many years and we are comfortable using it, especially when you have repositories in a tree of directories.   I find this basic concept lacking in GitHub making it difficult to migrate. </p>
<p>I have 150+ Visual Studio project, solutions in a tree stucture.  Each folder has its own CVS repository.   I don’t want to be “Gitting” the entire tree.  I have exhausted how to organized this without first have a linear model of 150+ repositories and then somehow  using Organizations to structure it like a tree.  I expect to be hiring about developers this year and I need to be able to assign/sandbox speciflc projects to individuals. I have a CVS layout like so:</p>
<p>\tools<br>
\lib3\openssl<br>
\local\wc{version}{project}[{sub-projects}]<br>
\build\parts\setup{version}\images<br>
\autoupdate{version}\images</p>
<p>Each folder has a DESCRIBE file. It is from the old APL WorkSpace concept.  There is also  a VERSION.NEW file in each folder to be aggregated during builds to document the history. </p>
<p>It would be easy if git was designed to offer sub-git models.  I don’t like the idea we have to fit into GitHub model rather than GitHub have models that fit into the wide ranges of structures.  </p>
<p>Suggestions, methods and tips would be appreciated.</p>
<p>Thanks</p>

**Comment by yamiacat** [comment, marked as answer]
<p>Hi hlsantos!</p>
<p>Unfortunately the GiHub model is indeed to have a linear list of repositories, and control access via teams. It is possible to nest teams to create a tree like structure:</p>
<p><a href="https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/about-teams#nested-teams" class="onebox" target="_blank" rel="nofollow noopener">https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/about-teams#nested-teams</a></p>
<p>The product team are very interested in making GitHub organizations more usable for an many people as possible, so they’d welcome any specific feedback or feature requests you might want to make about org accounts!</p>
<p><a href="https://support.github.com/contact/feedback" class="onebox" target="_blank" rel="nofollow noopener">https://support.github.com/contact/feedback</a></p>
